components/FeaturedStories/FeatureMasonry.tsx

```diff
@@ -80,7 +82,7 @@ export const FeatureMasonry = ({
               quoteColor="flamingo"
               body={quoteBody}
               isLargeBody
-              source="Eden Hadar, ’26"
```

Comment:
They have only use this component on one story (the original plan was that it was just for this story). I made some attempt to make it available for general usage, but I missed this bit 🤦🏼‍♀️